website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-enterprise-sso-with-azure-active-directory.md

```diff
@@ -78,11 +79,23 @@ the Redirect URI in the steps above, then skip ahead to step 8.
 
 <Lightbox collapsed="true" src="/img/docs/dbt-cloud/dbt-cloud-enterprise/azure/azure-redirect-uri.png" title="Configuring a Redirect URI"/>
 
+### Adding Users to an Enterprise Application
+
+Once you've registered the application, the next step is to assign users to it. Add the users you want to be viewable to dbt with the following steps:
+
+8. From the **Default Directory** click **Enterprise Applications**
+9. Click the name of the application you created earlier
+10. Click **Assign Users and Groups**
+11. Click **Add User/Group**
+12. Assign additional users and groups as-needed
+
+# Where do I go to upload the image as referenced in the path below? Not seeing this from navigating this git repo or searching for /img/ as a repository at the top level of the dbt org.
```

Comment:
Images are served from the [website/static/](https://github.com/fishtown-analytics/docs.getdbt.com/tree/current/website/static) directory. So, the image above points to `/img/docs/dbt-cloud/dbt-cloud-enterprise/azure/azure-redirect-uri.png` which is [this file]((https://github.com/fishtown-analytics/docs.getdbt.com/blob/current/website/static/img/docs/dbt-cloud/dbt-cloud-enterprise/azure/azure-redirect-uri.png)

Reply:
@drewbanin FYI fixed the image links

Reply:
@matt-winkler are those image changes in this PR? https://github.com/fishtown-analytics/docs.getdbt.com/pull/500

Would be good to merge these two branches together IMO, but LGTM once the images are being rendered in dev/staging :)